spotifycli/spotifycli.py

```diff
@@ -172,13 +204,15 @@ def show_song_short():
 
 
 def show_lyrics():
+    artist, title = get_song()
     try:
-        artist, title = get_song()
         lyrics = lyricwikia.get_all_lyrics(artist, title)
-        lyrics = ''.join(lyrics[0])
-        print(lyrics)
-    except BaseException:
-        print('lyrics not found')
+    except lyricwikia.LyricsNotFound:
+        raise SpotifyCLIException(
+            'Lyrics not found or could not connect'
+        ) from None
```

Comment:
This is an example of a "known" issue.  The lyrics engine may throw a failure if it either cannot connect or cannot find the lyrics.  It appears in either case a LyricsNotFound is thrown.  The user doesn't really need to see a python traceback to understand what to do.  So instead, we handle that exception, and instead throw a new SpotifyCLIException to explain the issue in a simple phrase.

Note that the `from None` is needed on the end of the new exception so that it clears the exception context and doesn't throw a nested exception response.